sql/updates/reverse-dev.sql

```diff
@@ -3,3 +3,6 @@ GRANT ALL ON _timescaledb_internal.job_errors TO PUBLIC;
 ALTER EXTENSION timescaledb DROP VIEW timescaledb_information.job_errors;
 
 DROP VIEW timescaledb_information.job_errors;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.ping_data_node(node_name NAME) RETURNS BOOLEAN
```

Comment:
I believe we will need to DROP the new API first and then create with older signature?